.changeset/proud-ghosts-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
+---
+
+Remove unused iframe rubric type
```

Comment:
I don't see any changes to our main package file (`index.ts`) which means that this type was not exported from Perseus, so a `patch` release sounds right to me.